returnn/tf/hyper_param_tuning.py

```diff
@@ -635,6 +635,7 @@ def run(self):
     for p in self.optim.hyper_params:
       print(" %s -> %s" % (p.description(), hyper_param_mapping[p]), file=log.v2)
     config = self.optim.create_config_instance(hyper_param_mapping, gpu_ids=self.gpu_ids)
+    config.set("task", "train")
```

Comment:
Why is that needed? That is already the default.

Reply:
here it is set to `"hyper_param_tuning"`

Reply:
But why is that relevant?

Reply:
You should add a comment to this why it is relevant.
Sth like:
```suggestion
    # init_train_from_config expects to have the train task
    config.set("task", "train")
```